### PR TITLE
Fix: Choose controls are not working as expected [ED-7373]

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

File: assets/dev/js/editor/controls/dimensions.js [hunk 1]
```diff
@@ -144,6 +144,15 @@ ControlDimensionsItemView = ControlBaseUnitsItemView.extend( {
...
 
+		// When using input with type="number" and the user start typing `-`, the accual value (event.target.value) is
+		// empty string, so to prevent updating the input again with empty string and the user has intention to write
+		// negative value the methods below will not be triggered.
+		const hasIntentionForNegativeNumber = '-' === event?.originalEvent?.data && ! event.target.value;
+
+		if ( hasIntentionForNegativeNumber ) {
+			return;
+		}
+
```
Comment (StyleShit) [inline assets/dev/js/editor/controls/dimensions.js:155]: what about other numeric contorls?

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/controls/dimensions.js (hunk 1 above)
Comment (Nevoss) [reply assets/dev/js/editor/controls/dimensions.js:155]: NO 😉

**Review by StyleShit — DISMISSED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]
Please cover with tests

File: package.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
         "test:qunit": "grunt qunit",
         "test:playwright": "playwright test \"--config=tests/playwright/config/sanity.config.js\"",
         "test:playwright:debug": "npm run test:playwright -- --debug",
-        "dev:start": "npm install && composer install",
+		"test:playwright:local": "playwright test \"--config=tests/playwright/config/local.config.js\"",
```
Comment (udidol) [inline package.json:15]: Maybe add a comment about this in the task?
I will add a comment about it in the commit message when this PR is squashed and merged.

**Review by udidol — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (Nevoss) [reply package.json:15]: Done

File: package.json [hunk 3]
```diff
@@ -12,7 +12,8 @@
...
+		"dev:start": "npm install && composer install",
```
[the remark this answers is not in the document]
Comment (Nevoss) [reply package.json:16]: I changed it back to spaces, seems like all the `package.json` uses spaces and not tabs, don't know why.

Comment (Nevoss) [inline package.json:16]:
```suggestion
        "test:playwright:local": "playwright test \"--config=tests/playwright/config/local.config.js\"",
        "dev:start": "npm install && composer install",
```

**Review by udidol — DISMISSED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]

**Pull request merged**